### ISSUE TITLE
Request of file.AsyncWrite and file.AsyncAppend

### ISSUE THREAD

**Issue by trust-paulfr** [Facepunch/garrysmod-requests#2199]
### Details

It's the same of file.Write with asynchronous 
Writes the given string to a file in async

```lua
file.AsyncWrite(fileName, content, callback, sync)
```

_And the same with file.AsyncAppend_

**Comment by FlorianLeChat** [comment]
Until native support is available, here are two binary modules which add that function:
- https://github.com/WilliamVenner/gm_async_write (also adds `file.AsyncAppend` which could also be added to GMod)
- https://github.com/Pika-Software/gm_asyncio

**Comment by paulfuriox** [comment]
Yes, I saw them but I need the function on the client side and especially natively 🥲
